### PR TITLE
Display error when AV stream has expired

### LINKED ISSUE #4796: Hide player and display error where AV and audio stream is expired
**Is your feature request related to a problem? Please describe.**
Users in CPS can disable a stream so that it is no longer active on a MAP and cannot play content. This is because the channels are shared across services and editorial staff need to ensure they disable a stream where their content is no longer being streamed to a channel. 

We need to stop the media from playing when the stream is tagged as expired. 

**Describe the solution you'd like**
**Scenario: Unavailable Stream** 
Given a live stream has been flagged as expired in CPS
and the Ares response returns available = false
when the page is published 
then the stream will not be played and an error will be displayed stating that the video is unavailable

This MAP has an expired stream: https://test.bbc.com/pidgin/media-23256549
We need to ensure users cannot play the stream. This is the current pal example: https://pal.test.bbc.com/pidgin/media-23256549?_x_candy_override=https%3A%2F%2Fapi.test.bbc.co.uk

For AV, there will be a placeholder image (when provided by Ares) and the message "Sorry we are unable to bring you this content right now".

For Audio, there will be no placeholder image and the message "Sorry we are unable to bring you this content right now".

We will implement the error message and designs here: 
https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5e2b162bafdbbf9579e33625

We will default to English for now until we have translations for all services.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves https://github.com/bbc/simorgh/issues/4796

**Overall change:** 
Display error message and 'Content not available' message to the user when a live stream has expired.

**Code changes:**

- Render Message prop when available is set to false
- Updated props for Canonical Media Player and AMP

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/CpsAssetMediaPlayer/index.jsx [hunk 1]
```diff
@@ -38,13 +39,22 @@ const Wrapper = styled(GridItemConstrainedLarge)`
 const CpsAssetMediaPlayerContainer = ({ blocks, assetUri }) => {
   if (!assetUri) return null;
 
+  const mediaBlock = filterForBlockType(blocks, 'aresMedia');
+  const metadataBlock = filterForBlockType(
+    mediaBlock.model.blocks,
```
Comment (ryanmccombe) [inline src/app/containers/CpsAssetMediaPlayer/index.jsx]:
```suggestion
    path(['model', 'blocks'], mediaBlock),
```

File: src/app/containers/CpsAssetMediaPlayer/index.jsx [hunk 2]
```diff
@@ -38,13 +39,22 @@ const Wrapper = styled(GridItemConstrainedLarge)`
...
+    'aresMediaMetadata',
+  );
+
+  const { available } = metadataBlock.model;
```
Comment (ryanmccombe) [inline src/app/containers/CpsAssetMediaPlayer/index.jsx]:
```suggestion
  const available = path(['model', 'available'], metadataBlock);
```

**Review by ryanmccombe — APPROVED** [review]
👍

File: src/app/containers/CpsAssetMediaPlayer/index.jsx (hunk 1 above)
Comment (rebeccamcginn) [reply src/app/containers/CpsAssetMediaPlayer/index.jsx]: Thanks :)

File: src/app/containers/MediaPlayer/index.jsx [hunk 3]
```diff
@@ -112,7 +115,30 @@ const MediaPlayerContainer = ({
     translations,
   );
 
+  const landscapeRatio = '56.25%'; // (9/16)*100 = 16:9
+  const portraitRatio = '177.78%'; // (16/9)*100 = 9:16
+  const StyledMessageContainer = styled.div`
+    padding-top: ${({ portrait }) =>
+      portrait ? portraitRatio : landscapeRatio};
```
Comment (ryanmccombe) [inline src/app/containers/MediaPlayer/index.jsx]: `portrait` is never true so this code is unused - think it's throwing the test coverage numbers too :)

**Review by karinathomasbbc — APPROVED** [review]
Do we need translations for the 'content is not available' message, or is this a separate task?

**Comment by rebeccamcginn** [comment]
> Do we need translations for the 'content is not available' message, or is this a separate task?

We have the translations but I don’t believe we have yet made an issue for the work 🙂

**Pull request merged**